testutils/env.go

```diff
@@ -30,7 +31,7 @@ func SkipUnlessInternet(tb testing.TB) {
 	tb.Helper()
 	if internetConnected == nil {
 		var connected bool
-		conn, err := net.Dial("tcp", "mozilla.org:80")
+		conn, err := net.DialTimeout("tcp", "mozilla.org:80", 5*time.Second)
```

Comment:
[driveby] this should probably use `DialTimeout` if you're trying to check internet connectivity. Tests will hang in the absence of internet connection otherwise.